### PR TITLE
Allowing multiple homes

### PR DESCRIPTION
**Pull request by thorbaert**
This will allow multiple homes.
Also added to descriptions on !help

commands added:
`!home set homeName`
`!home homeName`
`!home list`
`!home delete homeName`

### CODE REVIEW

File: app/commands/home.js [hunk 1]
```diff
@@ -22,34 +31,71 @@ module.exports = Base => class extends Base {
...
+        await this.deletePlayerHome(playerName, homeName);
+        this.tellPlayer(playerName, 'Home deleted', 'green');
     }
 
-    async setHome(playerName) {
+    async listHomes(playerName) {
+        let homes = await this.readPlayerHomeList(playerName);
+
+        if (!homes) {
+            this.tellPlayer(playerName, 'You have not homes.');
```
Comment (Gobsmack90) [inline app/commands/home.js]: typo "you have _not_ homes"

**Review by Gobsmack90 — CHANGES_REQUESTED** [review]
maybe i just missed it, but it seems like this doesn't currently allow you to overwrite homes, so if i want to move my ```!home mine``` i couldn't just type ```!home set mine``` i'd have to type ```!home delete mine``` then ```!home set mine``` which i know isn't that big a deal, but is a quality of life feature i'd appreciate, also on that note, what happens if you have two homes with the same name?

File: app/commands/home.js (hunk 1 above)
Comment (thorbaert) [reply app/commands/home.js]: Thanks! fixed

**Comment by thorbaert** [comment]
@Gobsmack90 I can't reply  to you comment but this may help.

This chunk of code allows you to overwrite an existing home. When you set a home if it already exists it overwrites it here

`data/index.js` lines 58 - 67
```
                if (player.homes[homeName] || homeEntries.length < 3) {
                    player.homes = {
                        ...player.homes,
                        [homeName]: { pos: pos, rot: rot, world },
                    };
                    return player.save((err) => {
                        if (err) return reject(err);
                        resolve(player);
                    });
                }
```

**Comment by Gobsmack90** [comment]
oh i didn't quite understand what that was doing before. I like it. lets push this baby live.

**Review by Gobsmack90 — APPROVED** [review]

**Pull request merged**